regnety/dataset/imagenet.py

```diff
@@ -49,6 +52,8 @@ def __init__(
         image_size: int = 512,
         augment_fn: Union[str, Callable]  = "default",
         num_classes: int = 1000,
+        percent_valid: int = 1,
+        color_jitter: bool = False, 
         scale_to_unit: bool = True
```

Comment:
Space in between arguments and values.

Reply:
Didn't follow for args with type hints due to [this](https://google.github.io/styleguide/pyguide.html#36-whitespace).

Reply:
Ah I see, yes. Makes sense.